extensions/wikia/SpecialPageViews/SpecialPageViews.i18n.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+$messages = [];
+
+/** English (en)
+ * @author Adam Karmiński
+ */
+$messages['en'] = [
+	'special-pageviews-special-desc' => 'An extension to visually present data on page views number for a given wikia',
+	'special-pageviews-title' => 'Page views graph',
+	'special-pageviews-report-title' => 'Page views for this wikia',
+	'special-pageviews-report-serie' => 'Page views',
+	'special-pageviews-beta-info' => 'This is a beta version of the dashboard.',
+	'special-pageviews-redraw-link-text' => 'Redraw the chart',
+];
+
+/** Message documentation
+ * @author Adam Karmiński
+ */
+$messages['qqq'] = [
+	'special-pageviews-special-desc' => '{{desc}}',
+	'special-pageviews-title' => 'A title of the special page',
```

Comment:
Using "the" suggests that it's about this specific SpecialPage. And it probably won't be translated anyway since we use it as a beta on 10~20 communities. :)